app/frontend/src/features/constants.ts

```diff
@@ -36,6 +37,9 @@ export const WHY = "Why I use Couchers";
 export const SECTION_LABELS = {
   about: ABOUT_ME,
   home: HOME,
+  references: undefined, // REFERENCES,
+  photos: undefined, // PHOTOS,
+  favorites: undefined, // FAVORITES,
```

Comment:
```suggestion
```
I think these are leaving behind "ghost tabs" with empty labels and causing errors (since there aren't users called "references", "photos" etc) if the user accidentally clicked on what looks like whitespace next to the "home" tab.

Reply:
Ah that's possible, that was why I filtered out undefined values in the tabbar. I checked the tabbar after and didn't see a problem, but it makes sense the tabs are still there but invisible. I'll remove them.